hugo/content/_index.en.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Visitors
+---
+
+{{< message >}}
+_Farmers, nobels, citizens and travelers of all sort_ \
+_You are all formally invited to enter the Fantasy Court._
+{{< /message >}}
+
+{{< aside >}}
+    {{< banner rotate="-3" >}}
+        <h2>Want to be a vendor?</h2>
+        <p>
+            Check out our <a href="/standhouders">vendors-page</a>
+        </p>
+    {{< /banner >}}
+{{< /aside >}}
+
+# What is Fantasy Court?
+Fantasy Court is a fantasy festival that is focused on the somewhat unique and niche culture that surrounds fantasy events and LARP (Live Action Role Playing). This festival has a vast array of stalls, associations, and activities that all fall under the purview of the awesome world of fantasy. This would entail LARP associations, surprising board games, exotic foods, a LARP sword fighting tournament (with a special children’s division!), storytellers, a LARP archery range as well as a wide variety of stalls that sell anything related. We offer all that belongs in the world of fantasy, and anything that will make you feel part of that world.
+
+Most importantly, everyone is welcome! If you’re a student or not, a parent with children or alone; it does not matter who you are. You will be welcome and you will find something to do. There will be workshops organized, various activities, and the all impressive sword fighting tournament for children and adults!
+
+Do you want to be informed on all developments? Check out our [Facebook Page](https://www.facebook.com/FantasyCourt/).
+
+{{< aside >}}
+    {{< banner rotate="1" >}}
+        <h2>We are hosted by</h2>
+        {{< figure src="/images/Knights_logo.png" alt="ESRG Knights of the Kitchen Table Logo" >}}
+        <p>ESRG Knights of the Kitchen Table</p>
+    {{< /banner >}}
+{{< /aside >}}
+
+# Edition 2020, CANCELLED
+With pain in our hearts, we unfortunately needed to cancel Fantasy Court 2020. The current nationwide regulations concerning COVID-19 are in place until (at least) the 1st of June, whilst Fantasy Court 2020 would be on the 7th of June.
```

Comment:
Misschien even naar verleden tijd omschrijven:

The nationwide regulations concerning COVID-19 at the time did not allow for Fantasy Court 2020 on the 7th of June.